package.json

```
@@ -32,26 +32,26 @@
   "type": "module",
   "main": "dist/index.cjs",
   "module": "dist/index.js",
-  "types": "index.d.ts",
-  "typesVersions": {
-    ">=5.0": {
-      "*": [
-        "dist/types/*"
-      ]
-    }
-  },
   "exports": {
```

node10 doesn’t understand exports so it will not resolve without typesVersions.
types field is not required.

```suggestion
  "typesVersions": {
    ">=5.0": {
      ".": [
        "./dist/index.d.ts"
      ],
      "locale/*": [
        "./dist/locale/*.d.ts"
      ]
    }
  },
  "exports": {
```